pkg/kubectl/cmd/util/factory.go

```diff
@@ -165,6 +166,24 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 			if cfg.GroupVersion != nil {
 				cmdApiVersion = *cfg.GroupVersion
 			}
+			client, err := clients.ClientForVersion(&unversioned.GroupVersion{Version: "v1"})
+			if err != nil {
+				fmt.Printf("Failed to create client: %v", err)
+				return nil, nil
+			}
+			list, err := client.ThirdPartyResources(api.NamespaceDefault).List(api.ListOptions{})
```

Comment:
I thought third part resources lived under extensions.  Why are we looking for them with a client we negotiated at v1?